<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>661</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	border-radius:10px;
	border:2px solid rgb(51, 51, 51);
	font: 10pt &quot;Comic Sans MS&quot;;
}

QLineEdit{
	border-radius:10px;
	border:2px solid rgb(51, 51, 51);
}

QPushButton:focus {
	border:3px solid rgb(51, 51, 51);
}

QPushButton:hover {
	border:3px solid rgb(51, 51, 51);
}

QPushButton:pressed{
	border-radius:10px;
	border:2px solid rgb(51, 51, 51);
	background-color: rgb(73, 161, 255);
}</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>571</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>881</width>
     <height>571</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
	background-color: qlineargradient(spread:pad, x1:0.54, y1:0, x2:0.517045, y2:1, stop:0 rgba(0, 0, 0, 54), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(93, 93, 93);
</string>
    </property>
    <property name="text">
     <string>Enter Authentcation Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUsername">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsername">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius:20px;
}


QLineEdit:focus {
border:3px solid rgb(51, 51, 51);

}
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius:20px;
}


QLineEdit:focus {
border:3px solid rgb(51, 51, 51);

}
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblError">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
   </widget>
   <widget class="QLabel" name="DialogTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>7</y>
      <width>653</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Residential Area
Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>btnLogin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>txtUsername</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLogin</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLogin</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
